crates/uv/tests/pip_tree.rs

```diff
@@ -1500,3 +1499,248 @@ fn package_flag() {
     "###
     );
 }
+
+#[test]
+fn emit_version_specifier_simple() {
+    let context = TestContext::new("3.12");
+
+    let requirements_txt = context.temp_dir.child("requirements.txt");
+    requirements_txt.write_str("requests==2.31.0").unwrap();
+
+    uv_snapshot!(context
+        .pip_install()
+        .arg("-r")
+        .arg("requirements.txt")
+        .arg("--strict"), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Resolved 5 packages in [TIME]
+    Prepared 5 packages in [TIME]
+    Installed 5 packages in [TIME]
+     + certifi==2024.2.2
+     + charset-normalizer==3.3.2
+     + idna==3.6
+     + requests==2.31.0
+     + urllib3==2.2.1
+    "###
+    );
+
+    uv_snapshot!(context.filters(), context.pip_tree().arg("--emit-version-specifier"), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    requests v2.31.0
+    ├── charset-normalizer v3.3.2 [required: <4, >=2]
```

Comment:
conflicting feelings towards the space after the comma - curious what others think

Reply:
This is the canonical display for version specifiers 🤷‍♀️